### ISSUE TITLE
Inline sql statement fails when comparing the result with null

### ISSUE THREAD

**Issue by EotT123** [manifold-systems/manifold#561]
**Describe the bug**
Inline sql statement fails when comparing the result with null.

This compiles:
` Test1 result = "[.sql/]SELECT * FROM test1 LIMIT 1".fetchOne();`

while this doesn't:
`boolean result2 =  "[.sql/]SELECT * FROM test1 LIMIT 1".fetchOne() == null;`
with the following error:
```
  cannot find symbol
  symbol:   method fetchOne()
  location: class java.lang.String`
```

It does work when extracting the query to a separate `.sql` file.

**Desktop (please complete the following information):**
- OS Type & Version:  Windows 10
- Java/JDK version: 21
- IDE version (IntelliJ IDEA or Android Studio): IntelliJ IDEA 2023.3.6
- Manifold version: 2024.1.10
- Manifold IntelliJ plugin version: 2023.3.20

[Sample project available](https://github.com/EotT123/manifold-test/blob/master/src/main/java/test/Main.java#L20)

**Comment by rsmckinney** [comment]
Fix available with release 2024.1.11

**Comment by EotT123** [comment]
Thanks! It works.
